src/Sylius/Behat/Page/Shop/Checkout/AddressPage.php

```diff
@@ -259,24 +305,53 @@ private function isSignInActionAvailable()
         });
     }
 
+    /**
+     * @return bool
+     */
     private function waitForLoginAction()
     {
         $this->getDocument()->waitFor(5, function () {
             return !$this->hasElement('login_password');
         });
     }
 
+    /**
+     * @return bool
+     */
     private function waitForShippingProvinceSelector()
     {
         return $this->getDocument()->waitFor(5, function () {
             return $this->hasElement('shipping_country_province');
         });
     }
 
+    /**
+     * @return bool
+     */
     private function waitForBillingProvinceSelector()
     {
         return $this->getDocument()->waitFor(5, function () {
             return $this->hasElement('billing_country_province');
         });
     }
+
+    /**
+     * @return bool
+     */
+    private function waitForShippingProvinceInput()
+    {
+        return $this->getDocument()->waitFor(5, function () {
+            return $this->hasElement('shipping_province');
+        });
+    }
+
+    /**
+     * @return bool
+     */
+    private function waitForBillingProvinceInput()
+    {
+        return $this->getDocument()->waitFor(10, function () {
+            return $this->hasElement('billing_province');
```

Comment:
Can't we have a single `waitForElement` method and pass timeout as well as element code?